### PR TITLE
Update Zipkin exporter error handling

### PR DESCRIPTION
**Pull request by ChrisJBurns**
The Zipkin exporter specification for error handling has slightly changed regarding the removal of the `otel.status_description` tag in favour for the error tag.

Due to this, we now move the `error` tag setting into the `!status.isUnset()` condition.

Resolves [2272](https://github.com/open-telemetry/opentelemetry-java/issues/2272)

### CODE REVIEW

File: exporters/zipkin/src/main/java/io/opentelemetry/exporter/zipkin/ZipkinSpanExporter.java [hunk 1]
```diff
@@ -158,6 +157,10 @@ static Span generateSpan(SpanData spanData, Endpoint localEndpoint) {
     return spanBuilder.build();
   }
 
+  private static String getValueOrReturnEmpty(String value) {
```
Comment (anuraaga) [inline exporters/zipkin/src/main/java/io/opentelemetry/exporter/zipkin/ZipkinSpanExporter.java:160]:
```suggestion
  private static String nullToEmpty(String value) {
```

Just small nit, we're not really "getting the value" from anything here

**Review by anuraaga — COMMENTED** [review]
Thanks! Can you add a test with null description?

**Comment by ChrisJBurns** [comment]
@anuraaga Great minds think alike. I noticed the missing null test just after I committed (just waiting for it to build).

**Review by anuraaga — APPROVED** [review]
Thanks!

**Review by jkwatson — APPROVED** [review]
Thanks!
